sys/include/net/dhcpv6/client.h

```diff
@@ -170,6 +199,46 @@ void dhcpv6_client_conf_prefix(unsigned netif, const ipv6_addr_t *pfx,
                                unsigned pfx_len, uint32_t valid,
                                uint32_t pref);
 
+/**
+ * @brief   Checks if the given network interface is configured
+ *          to use DHCPv6 IA_NA
+ *
+ * @param[in] netif     Network interface to check.
+ *
+ * @return  true, if the network interface is set up for IA_NA.
+ */
+bool dhcpv6_client_check_ia_na(unsigned netif);
+
+/**
+ * @brief   Configures a address lease that is provided by the server.
+ *
+ * @param[in] netif     Network interface the address was for.
+ * @param[in] addr      The assigned address.
+ *
+ * @return  true, if the address was configured successfully.
+ */
+bool dhcpv6_client_add_addr(unsigned netif, ipv6_addr_t *addr);
+
+/**
+ * @brief   Deprecates an existing address from an address lease.
+ *
+ * @param[in] netif     Network interface the address was for.
+ * @param[in] addr      The address to deprecate.
+ *
+ * @return  true, if the address was deprecated successfully.
+ */
+bool dhcpv6_client_deprecate_addr(unsigned netif, const ipv6_addr_t *addr);
```

Comment:
Same here: You never check the return value, so is it really needed?

Reply:
I changed the return type to `void` now in 6941fc305ce7c5a99cf8205f437d5f4ba2eb9ed9